crypto/dsa/ecdsa/ecdsa.go

```diff
@@ -0,0 +1,83 @@
+package ecdsa
+
+import (
+	"fmt"
+
+	"github.com/tbd54566975/web5-go/jwk"
+)
+
+const (
+	KeyType = "EC"
+)
+
+var AlgorithmIDs = map[string]bool{
```

Comment:
I would make this private, as it's an implementation detail and you've got a function to wrap it already.

Reply:
good point! done: https://github.com/TBD54566975/web5-go/commit/045c2a726ed9a080d8e149538468972d9eab4a6b